### PR TITLE
added docker-compose.yml file and fix commands

### PR DESCRIPTION
**Pull request by binarygeek119**
added docker-compose.yml file and fix commands

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+                        - driver: nvidia
+                          count: all
+                          capabilities:
+                              - gpu
+        stdin_open: true
+        tty: true
+        container_name: UltraSinger 
+        volumes:
+            - ./src/output:/app/src/output
+        image: ultrasinger
```
Comment (rakuri255) [inline docker-compose.yml:17]: use version from docker hub

```suggestion
        image: rakuri255/ultrasinger:latest
```

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: ultrasinger
+services:
+    ultrasinger:
+        deploy:
+            resources:
+                reservations:
+                    devices:
+                        - driver: nvidia
```
Comment (rakuri255) [inline docker-compose.yml:8]: Does this work on machinges without nvidia gpus?

**Review by rakuri255 — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: ultrasinger
```
Comment (rakuri255) [inline docker-compose.yml:1]: Do we need this name here? Isnt container_name enough?

Comment (rakuri255) [inline docker-compose.yml:1]: Should we add version?

```suggestion
---
version: "3.9"

name: ultrasinger
```

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+                    devices:
+                        - driver: nvidia
+                          count: all
+                          capabilities:
+                              - gpu
+        stdin_open: true
+        tty: true
+        container_name: UltraSinger 
+        volumes:
+            - ./src/output:/app/src/output
```
Comment (rakuri255) [inline docker-compose.yml:16]: I dont like that the user has to add src folder. 
Maybe ./ultrasinger/output? or just ./output ?

**Review by rakuri255 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (rakuri255) [reply docker-compose.yml:8]: I tried it on my gpu less NAS 😄 

> UltraSinger Starting Error response from daemon: could not select device driver "nvidia" with capabilities: [[gpu]]

File: docker-compose.yml (hunk 1 above)
Comment (binarygeek119) [reply docker-compose.yml]: done it now uses the docker image

File: docker-compose.yml (hunk 2 above)
Comment (binarygeek119) [reply docker-compose.yml]: i have made a second one for non gpu users

File: docker-compose.yml (hunk 3 above)
Comment (binarygeek119) [reply docker-compose.yml]: i have added this to the two docker-compose.yml files

Comment (binarygeek119) [reply docker-compose.yml]: done

File: docker-compose.yml (hunk 4 above)
Comment (binarygeek119) [reply docker-compose.yml]: i have adjested it just testing it now

Comment (binarygeek119) [reply docker-compose.yml]: it is now working with /output
you do need to add 
`-o /output` the the command now

File: docker-compose.yml (hunk 2 above)
Comment (binarygeek119) [reply docker-compose.yml]: its been tested with and without gpu support. it now works with both.
do note there are new two docker-compses
one for gpu and one for cpu

File: docker-compose.yml (hunk 3 above)
Comment (binarygeek119) [reply docker-compose.yml]: i did need to add a .env file to make this work.

**Comment by binarygeek119** [comment]
i believe it ready for review now

File: docker-compose.yml (hunk 4 above)
Comment (rakuri255) [reply docker-compose.yml]: why not `- ./output:/app/src/output` ? So we doesent need the `-o` command.

Comment (binarygeek119) [reply docker-compose.yml]: this is a oversite with me. i was having trouble getting the mount point right. i forgot to undo it

File: docker-compose.yml (hunk 3 above)
Comment (binarygeek119) [reply docker-compose.yml]: i removed it.

File: README.md [hunk 5]
```diff
@@ -283,16 +283,42 @@ run this command to build the docker
...
+`            -  /yourfolderpathhere:/app/UltraSinger/src/output`
+sample
+`            -  /mnt/user/appdata/UltraSinger:/output`
+note the blank space before the `-`
+formating is important here in this file.
 
 this will create and drop you into the docker.
 now run this command.
-`python3 UltraSinger.py -i file`
+`python3 UltraSinger.py -i file -o /output`
```
Comment (rakuri255) [inline README.md:315]: We dont need the output anymore. or?

**Review by rakuri255 — CHANGES_REQUESTED** [review]

**Review by rakuri255 — APPROVED** [review]

**Pull request merged**